codeforboston/maple
# Edit Profile settings button and tweaks

# Summary

#1178

Keep the current toggle on settings, but also add the toggle button to the profile under the "Edit Profile" button

flags().followOrg moved directly to Follow Button; set to true for dev, while remaining false for prod

# Checklist


![image](https://media.tenor.com/e4l0IBMkyXsAAAAd/batman-time-to-go-mobile.gif)

# Screenshots

![image](https://github.com/codeforboston/maple/assets/73559781/67a75a21-94cd-427c-9758-f6c5c60ba4df)

![image](https://github.com/codeforboston/maple/assets/73559781/61b114e0-b96e-4c76-ba55-02e1971051a3)

![image](https://github.com/codeforboston/maple/assets/73559781/80b88bfd-85ed-4b8c-a7ed-b961152281ca)

# Known issues

needs fresh pair of eyes to optimize for Arrowness

# Steps to test/reproduce

1. Log In
2. Navigate to View Profile
3. Check that "Make Public/Private" toggle button appears and is functional
  * optionally change screen sizes to check Mobile break point @ 768px 
4. Navigate to an Org that you are not the primary account user for( such as http://localhost:3000/profile?id=8P5Ar8NyAcNEVJMTexezXvRbept2 )
5. Check that "Edit Profile" and "Make Public/Private" buttons *do not* appear
  * optionally change screen sizes to check Mobile break point @ 768px
